app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,15 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
+    user = User.find_by(username: params[:session][:username])
+    if user
+      log_in user
+      redirect_to user_path(user.id)        
+    else
+      # flash.now[:danger] = 'Invalid email/password combination'
```

Comment:
Uncomment this flash error it should let the user know if he did something wrong.